api/controllers/routes.go

```diff
@@ -20,10 +20,12 @@ func NewRouter() (router *mux.Router) {
 	router.HandleFunc("/apps/{app}", api("app.delete", AppDelete)).Methods("DELETE")
 	router.HandleFunc("/apps/{app}/builds", api("build.list", BuildList)).Methods("GET")
 	router.HandleFunc("/apps/{app}/builds", api("build.create", BuildCreate)).Methods("POST")
+	router.HandleFunc("/apps/{app}/builds/import", api("build.import", BuildImport)).Methods("POST")
```

Comment:
Can this be part of POST /apps/{app}/builds ?

The former already has a few different input types (tarball, repo url, incremental index)